source/_integrations/iammeter.markdown

```diff
@@ -16,11 +16,8 @@ ha_codeowners:
   
 demo show:
 
-[http://ha.iammeter.com:18123](http://ha.iammeter.com:18123)
+![image-20200623101938530](https://leweidoc.oss-cn-hangzhou.aliyuncs.com/lewei50/img/iammeter/iammeterimage-20200623101938530.png)
```

Comment:
Screenshots must be included in this PR.

Reply:
> Screenshots must be included in this PR.

Would you help to check if I did it right this time?

Thanks!